vflow/vset.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, name: str, modules, module_keys: list = None,
         if not self._async and np.any([isinstance(mod, AsyncModule) for mod in modules]):
             self._async = True
         if isinstance(modules, dict):
-            self.modules = modules
+            self.vfuncs = modules
```

Comment:
Let's also change the init argument names from `modules` and `module_keys` to `funcs` and `func_keys`.

Reply:
Should I also change  `apply_modules` -> `apply_vfuncs` (in utils.py)?

Reply:
Yes, please. Let's replace all usage of `module` with `vfunc`.